catatonit.c

```diff
@@ -482,6 +482,11 @@ int main(int argc, char **argv)
 	if (argc < 1 && !run_as_pause)
 		bail_usage("missing program name");
 
+	char *kill_pgid_env = secure_getenv("CATATONIT_KILLPG");
```

Comment:
`secure_getenv` doesn't make sense in this context -- `secure_getenv` is only relevant for setuid binaries (and even then, it only really makes sense for general purpose libraries that might be used by a setuid binary).